src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -466,10 +464,13 @@ export default class AircraftController {
      *
      * @for AircraftController
      * @method removeAircraftModelFromList
-     * @param  {AircraftModel} aircraft the aircraft to remove
+     * @param  {AircraftModel} aircraftModel the aircraft to remove
      */
-    removeAircraftModelFromList(aircraft) {
-        this.aircraft.list = _without(this.aircraft.list, aircraft);
+    removeAircraftModelFromList(aircraftModel) {
+        console.warn('AircraftController.removeAircraftModelFromList() has been deprecated');
```

Comment:
so this is where my `new api` comment comes from.  The output says this has been deprecated, but I see we are calling `this._collection.remove(aircraftModel);` just below.  so is this method _really_ deprecated?  has it been replaced?  Will it be replaced?

If yes, then I would suggest that instead of doing the work with the collection here, this method would simply call the 'new' api method to do the work there.  Also, if this is going to be replace, I would do some finds to figure out where `.removeAircraftModelFromList()` is actually getting called and update those references as well.

Reply:
Honestly, I don't know. According to the git history, you wrote the code for this PR 😉, so I hoped you know what to do here.

Reply:
ahh, so this is my breadcrumb then.

ok, with that in mind, I'd assume there is another method somewhere that is actually adding aircraft to the collection and should be used instead of this one.